.github/workflows/release.yml

```diff
@@ -23,7 +26,22 @@ jobs:
       - name: Test
         run: npm run test
       - name: Release
+        uses: cycjimmy/semantic-release-action@v2
+        with:
+          semantic_version: 16
+          branches: |
+            [
+              'master',
+              'next',
+              {
+                name: 'beta',
+                prerelease: true
+              },
+              {
+                name: 'alpha',
+                prerelease: true
+              }
+            ]
```

Comment:
FWIW, these are already the default options for semantic-release per the [documentation](https://semantic-release.gitbook.io/semantic-release/usage/configuration#branches).

```
['+([0-9])?(.{+([0-9]),x}).x', 'master', 'next', 'next-major', {name: 'beta', prerelease: true}, {name: 'alpha', prerelease: true}]
```

No harm in being explicit, though.

Reply:
If they're the defaults, I'd rather just remove the config... this whole file is, I think, from when you created it for this branch initially. :)  I haven't really touched it.  What do you think we can remove here, the whole `branches` section from line 32 on?

Reply:
I'm also in favor of removing config when it's already the defaults. In this case, right, it'd be the entire `branches` section :) 

I wonder if it's also worth running the `release.yml` workflow for the `next-major` branch (line additions 6-8 in this file). Not sure if we'll ever actually run a `next-major` branch but might be good to still map the workflow to the default `branches` config.